packaging/opensuse-42.2/snapd.spec

```diff
@@ -164,15 +164,15 @@ make %{?_smp_mflags} -C cmd check
 # Install all the go stuff
 %goinstall
 # TODO: instead of removing it move this to a dedicated golang package
-rm -rf %{buildroot}/usr/lib64/go
-rm -rf %{buildroot}/usr/lib/go
+rm -rf %{buildroot}%{_libexecdir}64/go
```

Comment:
This should just be `%_libdir`, though this really should just moved to a dedicated golang subpackage...